src/modules/extendedBuilding/assets/expansions.ts

```diff
@@ -76,7 +76,7 @@ export default (LSSM: Vue, MODULE_ID: string, $m: $m): void => {
             expansion.querySelector('a[href*="extension_ready"]') ||
             successLabel
         ) {
-            label.classList.add('label-success');
+            label.classList.add('.label-danger');
```

Comment:
This would make all green labels red. Also the `.` is invalid!
Correct code for the complete Line would probably be:
```js
label.classList.add(expansion.querySelector('.label-danger') ? 'label-danger' : 'label-success');
```